static-analysis/autodispose-lint/src/main/kotlin/com/uber/autodispose/lint/AutoDisposeIssueRegistry.kt

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (C) 2018. Uber Technologies
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.uber.autodispose.lint
+
+import com.android.tools.lint.client.api.IssueRegistry
+import com.android.tools.lint.detector.api.CURRENT_API
+import com.android.tools.lint.detector.api.Issue
+import com.google.auto.service.AutoService
+
+@AutoService(IssueRegistry::class)
+class AutoDisposeIssueRegistry(override val issues: List<Issue> = listOf(AutoDisposeDetector.ISSUE)) : IssueRegistry() {
```

Comment:
might be worth putting the issues into it's own property and not into the constructor as possibly you'll get more lint checks and the line is already long

Reply:
Resolved with 49b8ccd9455b36cc785ea5aee0f29f3e3185a41d